src/main/kotlin/no/skatteetaten/aurora/boober/feature/S3Feature.kt

```diff
@@ -164,34 +164,75 @@ private fun Secret.createEnvVarRefs(properties: List<String> = this.data.map { i
         }
     }
 
-fun S3ProvisioningResult.createS3Secret(nsName: String, s3SecretName: String) = newSecret {
-    metadata {
-        name = s3SecretName
-        namespace = nsName
+fun List<S3ProvisioningResult>.createS3Secrets(nsName: String, appName: String) = map {
+    newSecret {
+        metadata {
+            name = "$appName-${it.bucketName.substringAfterLast("-")}-s3" // TODO: possible problem with bucketname that has "-" in its suffix
```

Comment:
Hvorfor må vi gjøre denne substringen her?

Reply:
Dette er for å få tak i suffixen i bucketName, altså den brukeren spesifiserer. Det er litt skjørt med denne subStringen, ettersom den begrenser bucketName suffix til å ikke kunne inneholde bindestrek. Det finnes noen alternativer:
Regex(litt mer robust i forhold til å finne suffixen)
Lagre suffixen i S3ProvisioningResult
Map med key som suffix og S3ProvisioningResult som value

Reply:
Fixed